+precice/@Participant/private/preciceGateway.cpp

```diff
@@ -0,0 +1,353 @@
+// Gateway MexFunction object
+#include "mex.hpp"
+#include "mexAdapter.hpp"
+#include <iostream>
+#include <sstream>
+#include <string_view>
+#include "precice/Participant.hpp"
```

Comment:
```suggestion
#include "precice/precice.hpp"
```
should use the main preCICE include.